rs/messaging/src/routing/stream_handler/tests.rs

```diff
@@ -352,7 +351,7 @@ fn induct_loopback_stream_reroute_response() {
             // `available_guaranteed_response_memory` does not keep track of gc'ing
             // the response @22 in the loopback stream after inducting it.
             assert_eq!(
-                available_guaranteed_response_memory + inducted_response_count_bytes as i64,
+                available_guaranteed_response_memory,
                 stream_handler.available_guaranteed_response_memory(&inducted_state),
             );
```

Comment:
Some of these comments will need to be updated. Probably also some in production code.